doc/ug/getting_started_fpga.md

```diff
@@ -36,6 +36,18 @@ $ ./meson_init.sh
 $ ninja -C build-out sw/device/boot_rom/boot_rom_export_fpga_nexysvideo
```

Comment:
make this 
```suggestion
$ ninja -C build-out all
```

And skip the section below.

Reply:
fixed!